packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -214,6 +215,15 @@ export abstract class FluidDataStoreContext
 	/** If true, this means that this data store context and its children have been removed from the runtime */
 	private deleted: boolean = false;
 
+	public get IFluidInternalReferenceInfo(): IFluidInternalReferenceInfo {
+		//* GC class maybe should cover this case?
```

Comment:
Yeah I would suggest that because we are losing the `unreferencedTimestamp` here.

Reply:
At some point I needed to add this to get the test to pass.  But looking at the code now I think it should work.  I'll try removing it again and see what's up.

Reply:
I had the same thought about losing unreferenced time.